packages/cli/src/models/network/NetworkAppController.js

```diff
@@ -148,6 +164,28 @@ export default class NetworkAppController extends NetworkBaseController {
     log.error(`Possible migration method 'migrate' found in contract ${contractClass.contractName}. Remember running the migration after deploying it.`);
   }
 
+  _fetchOwnedProxies(packageName, contractAlias, proxyAddress) {
+    const proxies = this.networkFile.getProxies({ 
+      package: packageName || (contractAlias ? this.packageFile.name : undefined),
+      contract: contractAlias, 
+      address: proxyAddress
+    })
+
+    if (_.isEmpty(proxies)) {
+      log.info('No owned contract instances that match were found');
+      return [];
+    }
+
+    const ownedProxies = proxies.filter(proxy => !proxy.admin || proxy.admin === this.appAddress);
+
+    if (_.isEmpty(ownedProxies)) {
+      log.info('No matching contract instances are owned by this application');
+      return [];
```

Comment:
I think there is no need to specify different returns giving the one above will work as you are expecting in case no proxies were found